packages/components/datepicker/src/Calendar/README.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Calendar'
+status: 'beta'
```

Comment:
If we want to show the `note` on the documentation page this should be set to Alpha, as we only check for Alpha and deprecated:
https://github.com/contentful/forma-36/blob/0b200eb7d815647873889df31368de9a9de94274/packages/website/components/PageContent/PageContentHeader.tsx#L61-L63

Reply:
Or we could create a new note for Beta components, just need a copy for that, and we can have the same styling in general

Reply:
Thanks, I was looking for that 👍🏼